### ISSUE TITLE
HIPRAND_STATUS_NOT_IMPLEMENTED 

### ISSUE THREAD

**Issue by jinz2014** [ROCm/hipRAND#76]
hiprandCreateGeneratorHost returns HIPRAND_STATUS_NOT_IMPLEMENTED for random number generators. For example,

rocm 5.6

```
int main(int argc, char *argv[]) {
  hiprandGenerator_t gen;

  hiprandRngType_t rng = HIPRAND_RNG_PSEUDO_MRG32K3A;

  HIPRAND_CHECK(hiprandCreateGeneratorHost(&gen, rng));

  ...
}
```

**Comment by Maetveis** [comment]
in https://github.com/ROCmSoftwarePlatform/hipRAND/pull/94 hipRAND now delegates to rocRAND for creating host generators on the AMD platform instead of always returning an error. The generators are in progress of being added to rocRAND.

**Comment by tcgu-amd** [comment]
@jinz2014 Brief update, the change mentioned earlier is merged to the latest ROCm 6.2 [here](https://github.com/ROCm/hipRAND/blob/develop/library/src/amd_detail/hiprand.cpp#L118)

Please upgrade to ROCm 6.2 to avoid this issue. This ticket will be closed.

Thanks!
